workflows/templates/thamos-advise-template.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: thamos-advise
+  annotations:
+    thoth-station.ninja/template-version: 0.1.0-dev
+  labels:
+    app: thoth
+    component: thamos-advise
+spec:
+  templates:
+    - name: thamos-advise
+      inputs:
+        parameters:
+          - name: "GITHUB_EVENT_TYPE"
+          - name: "GITHUB_CHECK_RUN_ID"
+          - name: "GITHUB_INSTALLATION_ID"
+          - name: "ORIGIN"
+          - name: "REVISION"
+          - name: "THOTH_HOST"
+        artifacts:
+          - name: repository
+            path: /mnt/inputs/artifacts/repository
+            git:
+              repo: "{{inputs.parameters.ORIGIN}}"
+              revision: "{{inputs.parameters.REVISION}}"
+      script:
+        command:
+          - python3
+        name: thamos-advise
+        image: "quay.io/thoth-station/thamos:v0.8.0-dev"
+        source: |
+          import os
+          import json
+          from pathlib import Path
+          from thamos.lib import advise_here
+          from thamos.config import config
```

Comment:
To configure logging and set up reporting (Sentry configuration needs to be supplied via env vars):

```python
from thoth.common import init_logging

init_logging()
```